### DISCUSSION TITLE
z/OS Open Source Guild Meeting #16 - Tuesday, July 23 at 10am EST

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#837] [Guild]
### Agenda

Join us **Wednesday, July 23 at 10am EST** for our **16th z/OS Open Source Guild Meeting**! We look forward to seeing you as we have a lot of great content to share with you!

The session will feature guest speakers @HarithaIBM, @DevonianTeuchter  and @KeplerBoyce as they discuss their most recent contributions to z/OS Open Tools.

More specifically, we will be covering the following topics:
* Updates to z/OS Open Tools
* Demos:
  * Kepler Boyce will be discussing the new zopen audit functionality - a way to notify users of open source CVEs in their installed zopen tools
  * Russell Wilson will discuss exciting new features in the zopen package manager
  * Haritha will be discussing Boost, now available through z/OS Open Tools
* Summary of the new tools released and announcements on updates to existing tools
* We'll end with a discussion into what's next, survey and Q/A

Once again, thank you all for the growing support and collaboration within our community!

### How to Join?
Join from the webinar link
https://ibm.webex.com/ibm/j.php?MTID=me19ca26197970de15c63318985200788

Webinar number (access code): 146 381 5103

* Tuesday, July 23 at 10am EST - [guild_16_invite.ics.zip](https://github.com/user-attachments/files/16333883/guild_16_invite.ics.zip)

### Recording/Slide/Poll/Chat Log
* [Open_Source_Guild_16_final.pptx](https://github.com/user-attachments/files/16351612/Open_Source_Guild_16_final.pptx)
* Recording: https://ibm.webex.com/webappng/sites/ibm/recording/2b2b7b682b2a103dbadbe21cc7fa5c34/playback (pass: 696UvgYv)

**Comment by IgorTodorovskiIBM** [comment]
Slides and recording are now available (above)

**Reply by v1gnesh** [reply]
Although GPG is required within zopen for download verification, users can stick to SSH for commit signing - https://www.youtube.com/watch?v=C7M6FaTg_zQ
